deploy/Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM ubuntu:bionic
+MAINTAINER soheil@gmail.com
+
+ENV CREATE_MYSQL_BASIC_USER_AND_DB true
+ENV MYSQL_USER_NAME dvwa
+ENV MYSQL_USER_DB dvwa
+ENV MYSQL_USER_PASS p@ssw0rd
+
+# Install packages
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && \
+		apt-get -y install git wget unzip supervisor apache2 mysql-server php php-mysqli php-gd libapache2-mod-php
```

Comment:
Can we have MariaDB rather than MySQL, there are loads of little issues with MySQL that make it a pain to support